test/TestProjects/CadlFirstTest/Generated/cadl.json

```diff
@@ -2,6 +2,32 @@
   "Name": "demo.helloworld",
   "Description": "",
   "ApiVersions": ["0.1.0"],
+  "Enums": [
+    {
+      "$id": "4",
+      "Name": "SimpleEnum",
+      "Description": "Simple enum",
+      "EnumValueType": "string",
```

Comment:
Enum is properly generated only for the case when value type is a string. Since we're using existing generator code for enums, it looks like we don't have services that have different value type.